data/schedule.json

```diff
@@ -91,7 +91,7 @@
                             "Title": "Understanding NLP - What's the feeling ?",
                             "Speaker": "Lakshya Sivaramakrishnan (Google)",
                             "Hall": "Hall 5",
-                            "Url": "#"
+                            "Url": "https:/in.pycon.org/2018/workshop-prerequisites.html"
```

Comment:
Use the section links as I mentioned here - https://github.com/pythonindia/inpycon2018/pull/214/files#r222246564

Reply:
The problem with this https://github.com/pythonindia/inpycon2018/pull/214#discussion_r222290135